src/main/java/nl/knaw/dans/ingest/core/service/DatasetEditor.java

```diff
@@ -122,6 +124,21 @@ Map<Integer, FileInfo> addFiles(String persistentId, Collection<FileInfo> fileIn
             var id = addFile(persistentId, fileInfo);
             result.put(id, fileInfo);
         }
+        if (!isMigration) {
+            var path = zipFileHandler.zipOriginalMetadata(deposit.getDdmPath(), deposit.getFilesXmlPath());
+            var checksum = DigestUtils.sha1Hex(new FileInputStream(path.toFile()));
+            var fileMeta = new FileMeta();
+            fileMeta.setLabel("original-metadata.zip");
+            var fileInfo = new FileInfo(path, checksum, fileMeta);
+            var id = addFile(persistentId,fileInfo);
+            result.put(id, fileInfo);
+            try {
+                Files.deleteIfExists(path);
+            }
+            catch (IOException e) {
+                log.error("Unable to delete zipfile {}", path, e);
```

Comment:
```suggestion
                log.warn("Unable to delete zipfile {}", path, e);
```

Since we are not going to fail, make this warning instead.